bin/MadGraph5_aMCatNLO/cards/production/14TeV/tHbj_TL_M1000GeV_W10p/tHbj_TL_M1000GeV_W10p_proc_card.dat

```diff
@@ -0,0 +1,19 @@
+sset group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
```

Comment:
remove this line as there is another import below that overwrites this one

Reply:
done!